config/environments/production.rb

```diff
@@ -76,6 +76,8 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
+  config.middleware.use Rack::Attack
```

Comment:
I cleans up the logs for development. 

Don't really need the log noise while I'm developing. 

Sorry I threw that in there, just felt compeled